elastic-buffer-sim/tests/Tests/Bittide/Simulate.hs

```diff
@@ -34,24 +34,24 @@ tests = testGroup "Simulate"
 fastPeriod :: PeriodPs
 fastPeriod = hzToPeriod 200e6
 
-clockConfig :: Ppm -> ClockControlConfig
+clockConfig :: Ppm -> ClockControlConfig 7
 clockConfig clockUncertainty = ClockControlConfig
   { cccPessimisticPeriod = speedUpPeriod clockUncertainty fastPeriod
   , cccSettlePeriod      = fastPeriod * 200
   , cccDynamicRange      = clockUncertainty * 2
   , cccStepSize          = 10
-  , cccBufferSize        = 128
+  , cccBufferSize        = d7 -- 2**7 ~ 128
```

Comment:
It seems like you'd wan to use tyVars here.

Reply:
Actually I'm not sure why this structure is here at all. It can use `defClockControlConfig`.

Reply:
But also, this is just used as a default clock config for all the tests, so hardcoding values is fine IMO.

Reply:
this will be removed in #175